src/components/MultiHopTrade/components/FiatRamps/FiatRampTradeFooter.tsx

```diff
@@ -229,6 +183,17 @@ export const FiatRampTradeFooter = ({
             onSubmit={handleSubmit}
           />
         )}
+
+        {selectedQuote && !isLoading && (
+          <Display.Mobile>
+            <Protocol
+              onClick={onOpenQuoteList || (() => {})}
```

Comment:
I think we want to memoize the noop:

```ts
const handleNoOp = useCallback(() => {}, [])
onClick={onOpenQuoteList || handleNoOp}
```

or just memoize the whole thing:

```ts
const handleProtocolClick = useCallback(() => {
  onOpenQuoteList?.()
}, [onOpenQuoteList])

onClick={handleProtocolClick}
```

Reply:
Or [`lodash/noop`](https://lodash.com/docs/4.17.15#noop)?